ui/pages/login.tsx

```diff
@@ -33,16 +33,21 @@ import {
 const Login: NextPage = () => {
   const [flow, setFlow] = useState<LoginFlow>();
   const [isSequencedLogin, setSequencedLogin] = useState(false);
+  const [baseURL, setBaseURL] = useState("");
```

Comment:
I don't think we need this. Can't we use `window.location.origin` below instead?

Reply:
this will omit the path of the URL, in juju deployments the baseURL will be `https://some/path/` and in local deployments it will be `http://localhost:4455`. IMO This is an easy workaround that allows us to be flexible

Reply:
I think we know which path we are on, because this file is login under the pages directory. If we strip the /login?... from the end of the current url, that should be a safe way to calculate this, while preserving paths in the base url.

Reply:
That's true, I still believe this is the most flexible approach though as it does not make any assumptions on the relative position of the paths, thus I prefer it. Do you see any problems with this?

Also I just realized that (most likely) the request to the `app-config` endpoint will not work on juju deployments, as the path `/api/v0/app-config` is wrong, it should leverage the base_url instead (cc @natalian98).

Reply:
I see no reason for it to be unreachable on juju, but I tried it to make sure and the endpoint is accessible:
```
$ curl http://10.64.140.43/test2-identity-platform-login-ui-operator/api/v0/app-config
{"oidc_webauthn_sequencing_enabled":false}
```

Reply:
@edlerd does [this](https://github.com/canonical/identity-platform-login-ui/blob/798330ab0210e4f1ed5ef763cfa20e4da2511fef/ui/pages/login.tsx#L40) call respect the base url?
On juju deployments it's similar to:
```
$ curl http://10.64.140.43/test2-identity-platform-login-ui-operator/api/v0/app-config
{"oidc_webauthn_sequencing_enabled":false}
```

Reply:
> @edlerd does [this](https://github.com/canonical/identity-platform-login-ui/blob/798330ab0210e4f1ed5ef763cfa20e4da2511fef/ui/pages/login.tsx#L40) call respect the base url? On juju deployments it's similar to:
> 
> ```
> $ curl http://10.64.140.43/test2-identity-platform-login-ui-operator/api/v0/app-config
> {"oidc_webauthn_sequencing_enabled":false}
> ```

I think that call will not work well on juju deployments.